src/api/resolvers/types/mutations/payloads/project/GraphQLAddComponentToProjectInput.ts

```diff
@@ -0,0 +1,29 @@
+import { GraphQLObjectTypeConfig, GraphQLObjectType, GraphQLString } from "graphql";
```

Comment:
That file name can't be right...

Reply:
fixed with fd7e8df8eefe9e3f7d97ed076de53856563cca0a